server/controllers/user.js

```diff
@@ -0,0 +1,50 @@
+import helpers from '../helpers';
```

Comment:
Please, could you rename this file to upper camel case as it's default export is a class?

Reply:
Done